### PR TITLE
ExtMessageHandler -> PipesMessageHandler; ExtResult --> PipesResult

### PR DESCRIPTION
**Pull request by schrockn**
## Summary & Motivation

See title. Those two renames. 

## How I Tested These Changes

BK

### CODE REVIEW

**Comment by schrockn** [comment]
Current dependencies on/for this PR:
* master
  * **PR #16776** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16776" target="_blank">Graphite</a> 
    * **PR #16777** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16777" target="_blank">Graphite</a> 
      * **PR #16778** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16778" target="_blank">Graphite</a> 
        * **PR #16779** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16779" target="_blank">Graphite</a> 
          * **PR #16780** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16780" target="_blank">Graphite</a> 
            * **PR #16781** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16781" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16781?utm_source=stack-comment).

File: python_modules/dagster/dagster/_core/pipes/context.py [hunk 1]
```diff
@@ -32,14 +32,14 @@
 from dagster._core.execution.context.invocation import BoundOpExecutionContext
 from dagster._core.pipes.client import PipesMessageReader
 
-ExtResult: TypeAlias = Union[MaterializeResult, AssetCheckResult]
+PipesResult: TypeAlias = Union[MaterializeResult, AssetCheckResult]
```
Comment (alangenfeld) [inline python_modules/dagster/dagster/_core/pipes/context.py:35]: this name feels weird to me. `PipesAssetResult` maybe?

**Review by alangenfeld — APPROVED** [review]
`PipesMessageHandler` is  good
`PipesResult` is meh, but isn't user facing outside of custom impls so its fine

**Comment by schrockn** [comment]
### Merge Activity

* **Sep 26, 2:40 PM**: @schrockn started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16781).

File: python_modules/dagster/dagster/_core/pipes/context.py (hunk 1 above)
Comment (schrockn) [reply python_modules/dagster/dagster/_core/pipes/context.py:35]: I want to keep this name generic to support the raw op use case. And there is no guarantee that we will _only_ have asset-based events going forward.

I'm going to land this as is and make a note of all names we need followups on.

**Pull request merged**